packages/terra-graphs/src/components/Bubble/BubbleSingleDataset.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import '../Graph.module.scss';
+import './BubbleSingleDataset.module.scss';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+
+const propTypes = {
+  /**
+   * ID for graph created.
+   */
+  graphID: PropTypes.string.isRequired,
+  /**
+   * Graph configuration object
+   */
+  graphConfig: PropTypes.object.isRequired,
+  /**
+   * data to be displayed in graph
+   */
+  dataset: PropTypes.array,
+  /**
+   * timeout to display multiple data contents in specific time interval.
+   */
+  timeout: PropTypes.array,
+};
+
+const BubbleSingleDataset = ({
+  graphConfig, dataset, graphID, timeout,
+}) => {
+  React.useEffect(() => {
+    const graph = Carbon.api.graph(graphConfig);
+    if (!(utils.isUndefined(dataset))) {
+      if (!(utils.isUndefined(timeout))) {
+        let count = 0;
+        dataset.forEach((data) => {
+          setTimeout(
+            () => (graph.graphContainer
+              ? graph.loadContent(Carbon.api.bubbleSingleDataset(data))
+              : ''),
+            timeout[count],
+          );
+          count += 1;
+        });
+      } else {
+        dataset.forEach((data) => {
+          graph.loadContent(Carbon.api.bubbleSingleDataset(data));
+        });
+      }
+    }
+  }, [graphConfig, dataset, timeout]);
+
+  return (
+    <div id="canvasContainer">
```

Comment:
We should avoid hard-coded IDs. IDs need to be unique to a page. If id's are needed they should either be passed in or dynamically assembled.